solutions/standard/variables.tf

```diff
@@ -0,0 +1,98 @@
+  type        = string
+  default     = "private"
+
+  validation {
+    condition     = contains(["public", "private", "public-and-private"], var.provider_visibility)
+    error_message = "Invalid visibility option. Allowed values are 'public', 'private', or 'public-and-private'."
+  }
+}
+
+########################################################################################################################
+# SCC variables
+########################################################################################################################
+
+variable "scc_workload_protection_instance_name" {
+  description = "The name for the Workload Protection instance that is created by this solution. Must begin with a letter. Applies only if `provision_scc_workload_protection` is true. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+  type        = string
+  default     = "base-security-services-scc-wp"
+}
+
+variable "scc_region" {
+  type        = string
+  default     = "us-south"
+  description = "The region to provision Security and Compliance Center resources in."
+}
+
+variable "scc_workload_protection_instance_tags" {
+  type        = list(string)
+  description = "The list of tags to add to the Workload Protection instance."
+  default     = []
+}
+
+variable "scc_workload_protection_resource_key_tags" {
+  type        = list(string)
+  description = "The tags associated with the Workload Protection resource key."
+  default     = []
+}
+
+variable "scc_workload_protection_access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the Workload Protection instance. Maximum length: 128 characters. Possible characters are A-Z, 0-9, spaces, underscores, hyphens, periods, and colons. [Learn more](https://cloud.ibm.com/docs/account?topic=account-tag&interface=ui#limits)."
+  default     = []
+
+  validation {
+    condition = alltrue([
+      for tag in var.scc_workload_protection_access_tags : can(regex("[\\w\\-_\\.]+:[\\w\\-_\\.]+", tag)) && length(tag) <= 128
+    ])
+    error_message = "Tags must match the regular expression \"[\\w\\-_\\.]+:[\\w\\-_\\.]+\", see https://cloud.ibm.com/docs/account?topic=account-tag&interface=ui#limits for more details"
+  }
+}
+
+variable "scc_workload_protection_service_plan" {
+  description = "The pricing plan for the Workload Protection instance service. Possible values: `free-trial`, `graduated-tier`."
+  type        = string
+  default     = "graduated-tier"
+  validation {
+    error_message = "Plan for Workload Protection instances can only be `free-trial` or `graduated-tier`."
+    condition = contains(
+      ["free-trial", "graduated-tier"],
+      var.scc_workload_protection_service_plan
+    )
+  }
+}
```

Comment:
You are missing `cbr_rules`. It will require a helper doc too and variable description should point to it (see example: https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/f13624fc46021113da327c4ff91146dc8422e622/solutions/instance/variables.tf#L147)